src/api/metrics_api.py

```diff
@@ -107,6 +107,9 @@
     generate_llm_summary,
 )
 from core.config import CHAT_SCOPE_FLEET_WIDE, FLEET_WIDE_DISPLAY
+from mcp_server.tools.observability_openshift_tools import (
```

Comment:
Can we use core functions instead of mcp-server? metrics-api should depend on core logic, not on mcp-server code.

Reply:
Good idea. Done.